org.eclipse.xtext.idea.junit/src/org/eclipse/xtext/xbase/idea/formatting/AbstractXbaseAdjustLineIndentTest.xtend

```diff
@@ -1947,12 +1951,14 @@ abstract class AbstractXbaseAdjustLineIndentTest extends LightToolingTest {
 			catch (java.lang.IllegalStateException e) return 2
 			catch (java.lang.Exception e) return 3
 			finally b = true
+			println(b)
```

Comment:
should be before try

Reply:
I thought it complains about the unused assignment.